Cimpress.Stereotype/StereotypeRequest.cs

```diff
@@ -52,6 +58,24 @@ public IStereotypeRequest SetExpectation(string contentType, decimal probability
             return this;
         }
 
+        public IStereotypeRequest SetWhiteList(string whiteListEntry)
+        {
+            _whiteListRels.Add(whiteListEntry);
+            return this;
+        }
+
+        public IStereotypeRequest SetBlackList(string blackListEntry)
```

Comment:
I "set" a good name here? Maybe simply `BlackList` as a verb... or `BlackListRelationship` or `AddBlackListEntry` ?

Reply:
Changed to `WithWhitelistedRelation`